i18n/zh/docusaurus-plugin-content-docs/version-v1.5/reference/karmadactl/karmadactl-usage-conventions.md

```diff
@@ -137,22 +137,9 @@ spec:
         end
     dependencyInterpretation:
       luaScript: >
+        local kube = require("kube")
```

Comment:
For such enhanced changes, I think only the current version needs to be changed.

Reply:
Yes, it's more reasonable.